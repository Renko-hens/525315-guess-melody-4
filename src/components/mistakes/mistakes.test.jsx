import React from "react";
import renderer from "react-test-renderer";
import Mistakes from "./mistakes.jsx";

describe(``, () => {
  it(`With noone count`, () => {
    const tree = renderer.create(
        <Mistakes
          count={0}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`With one count`, () => {
    const tree = renderer.create(
        <Mistakes
          count={1}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
