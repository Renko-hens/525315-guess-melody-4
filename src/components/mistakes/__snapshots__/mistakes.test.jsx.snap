// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With noone count 1`] = `
<div
  className="game__mistakes"
/>
`;

exports[`With one count 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
</div>
`;
